src/profiling/meter_manager.cpp

```diff
@@ -141,29 +148,38 @@ nlohmann::json to_json(const meter_report& report) {
 
 // Print easy to read report of meters to a stream.
 std::ostream& operator<<(std::ostream& o, const meter_report& report) {
-    o << "\n---- meters ------------------------------------------------------------\n";
-    o << strprintf("%21s", "");
+    o << "\n---- meters -------------------------------------------------------------------------------\n";
+    o << strprintf("meter%16s", "");
     for (auto const& m: report.meters) {
         if (m.name=="time") {
-            o << strprintf("%16s", "time (s)");
+            o << strprintf("%16s", "time(s)");
         }
         else if (m.name.find("memory")!=std::string::npos) {
-            o << strprintf("%16s", m.name+" (MB)");
+            o << strprintf("%16s", m.name+"(MB)");
+        }
+        else if (m.name.find("energy")!=std::string::npos) {
+            o << strprintf("%16s", m.name+"(kJ)");
```

Comment:
This is getting brittle and obscure: perhaps now is the time to store unit and scale information in the meter data...

Reply:
That is a PR for another day...